### PR TITLE
Stub out new localization.md for names including pgf

### PR DESCRIPTION
**Pull request by nvkelso**
Re https://github.com/protomaps/basemaps/pull/265/files#r1657793275

/cc @wipfli

### CODE REVIEW

**Comment by nvkelso** [comment]
@wipfli can you take a pass at editing this, please? Maybe add some examples in the MapLibre section at the bottom, too? Thanks :)

File: basemaps/localization.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+To facilitate Protomap's support of additional, non-supported scripts in MapLibre (like Hindi), Protomaps exports names with "positioned glphys" so MapLibre can use codepoints as indices of positioned glyphs in an additional custom "font stack". While the raw `pmap:pgf:name:*` values look like giberish when inspecting the raw values, they will render correctly in MapLibre.
+
+See more:
+
+- [Traditional MapLibre text rendering](https://github.com/wipfli/about-text-rendering-in-maplibre)
+- [Positioned glyph fonts](https://github.com/wipfli/positioned-glyph-font)
```
Comment (wipfli) [inline basemaps/localization.md:172]:
```suggestion
```
I plan to deprecate this repo and write a separate blog post instead...

File: basemaps/localization.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+Rendering text in web browsers works for almost all languages and scripts and feels like magic. However, specialized map renderers like MapLibre have to reimplement text rendering and text layout which is complicated when text needs to be curved along linear map features instead of placed only horizontally or vertically. MapLibre normally assumes a one-to-one mapping between glyphs and Unicode codepoints that also exist in MapLibre font files (aka "font stacks") to accomplish the layout for a large but limited number of scripts. Plugins have been developed to extend MapLibre for **right-to-left** scripts like Arabic and Hebrew, and for **CJK scripts** like Chinese, Japanese, and Korean.
+
+To facilitate Protomap's support of additional, non-supported scripts in MapLibre (like Hindi), Protomaps exports names with "positioned glphys" so MapLibre can use codepoints as indices of positioned glyphs in an additional custom "font stack". While the raw `pmap:pgf:name:*` values look like giberish when inspecting the raw values, they will render correctly in MapLibre.
```
Comment (wipfli) [inline basemaps/localization.md:167]:
```suggestion
To facilitate Protomap's support of additional, non-supported scripts in MapLibre (like Devanagari), Protomaps exports names with "positioned glphys" so MapLibre can use codepoints as indices of positioned glyphs in an additional custom "font stack". While the raw `pmap:pgf:name:*` values look like giberish when inspecting the raw values, they will render correctly in MapLibre.
```

**Comment by nvkelso** [comment]
@wipfli Is this docs PR still useful? It may need to be updated for a few recent changes...

**Comment by wipfli** [comment]
See https://github.com/protomaps/docs/pull/47 @nvkelso

**Comment by wipfli** [comment]
I would leave the document as it is right now and publish it. Once we have language localization in the TypeScript I can update the relevant sections in the document.

**Comment by bdon** [comment]
Merging this first! Thanks.

**Pull request merged**